docs/api-docs/getting-started/webhooks/webhook-events.md

```diff
@@ -557,7 +557,7 @@ Changes to any of the following fields will trigger a `store/product/updated` ev
 - Product Type
 - Sort Order
 - Tax Price
-- Thumbnail Changed
+- Thumbnail (new images only)

```

Comment:
One important note/distinction that was mentioned in the ticket is that "The only thing that fires the updated event is when you go from having no thumbnail at all to a thumbnail," but uploading/updating a new image does not trigger an event. I think we may need to explicitly mention this.